src/o365/spo/commands/userprofile/userprofile-get.spec.ts

```diff
@@ -0,0 +1,312 @@
+import commands from '../../commands';
+import Command, { CommandValidate, CommandOption } from '../../../../Command';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+const command: Command = require('./userprofile-get');
+import * as assert from 'assert';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import auth from '../../../../Auth';
+
+describe(commands.USERPROFILE_GET, () => {
+  let vorpal: Vorpal;
```

Comment:
Vorpal is no more required in the latest version of the CLI. If you update your solution you can get rid of it